README.md

```diff
@@ -51,6 +51,20 @@ git submodule update --recursive
 
 ### Build instructions
 
+To build graft_server run *cmake* and then *make* like following
+
+```bash
+cmake ..
+make
+```
+
+If you want to build graft_server_test also, run *cmake* with additional parameter
```

Comment:
"If you want to build unit test suit as well, run *cmake* with additional parameter:"
----------------------------
how do they run it?